docs/Reply.md

```diff
@@ -5,6 +5,7 @@ The second parameter of the handler function is `Reply`.
 Reply is a core Fastify object that exposes the following functions:
 
 - `.code(statusCode)` - Sets the status code.
+- `.status(statusCode`)` - An alias for `.code(statusCode)`.
```

Comment:
At the end of `statusCode` there is an additional backtick that is breaking the format.

Reply:
If you can fix it that would be great. Otherwise, it will have to wait until I am off work. Sorry for the mess; don't know how that got in there.